cmake/modules/OpenCL.cmake

```diff
@@ -50,9 +50,18 @@ endif(USE_AOCL)
 
 if(USE_OPENCL)
   if (NOT OpenCL_FOUND)
-    find_package(OpenCL REQUIRED)
+    find_package(OpenCL)
```

Comment:
are these changes done only for adding definition of OpenCL_INCLUDE_DIRS? in this case let's remove everything else related to finging of opencl (we do not need it) and related to linking to opencl

Reply:
This `find_package` also set `OpenCL_LIBRARIES` this variable specifies which OpenCL library should be linked. But agree with you that the system OpenCL library should be found [here](https://github.com/apache/tvm/pull/13362/files#diff-45747f6e0c43e6e7579d2c50160a4bdd25108cf04e98dc7ed8f1b36ddcad09b1R19). 
I will remove this condition.